dpnp/dpnp_algo/dpnp_elementwise_common.py

```diff
@@ -341,6 +341,105 @@ def __call__(
                 return out
             return dpnp_array._create_from_usm_ndarray(res_usm)
 
+    def outer(
+        self,
+        x1,
+        x2,
+        out=None,
+        where=True,
+        order="K",
+        dtype=None,
+        subok=True,
+        **kwargs,
+    ):
+        """
+        Apply the ufunc op to all pairs (a, b) with a in A and b in B.
+
+        Parameters
+        ----------
+        x1 : {dpnp.ndarray, usm_ndarray}
+            First input array.
+        x2 : {dpnp.ndarray, usm_ndarray}
+            Second input array.
+        out : {None, dpnp.ndarray, usm_ndarray}, optional
+            Output array to populate.
+            Array must have the correct shape and the expected data type.
+        order : {"C", "F", "A", "K"}, optional
```

Comment:
numpy support `order = None` also
```suggestion
        order : {None, "C", "F", "A", "K"}, optional
```